persistence-jdbc/src/main/scala/zio/actors/persistence/jdbc/JDBCJournal.scala

```diff
@@ -0,0 +1,77 @@
+package zio.actors.persistence.jdbc
+
+import cats.effect.Blocker
+import com.zaxxer.hikari.HikariDataSource
+import doobie._
+import doobie.hikari.HikariTransactor
+import doobie.implicits._
+
+import zio.{ DefaultRuntime, IO, Promise, Task, ZIO }
+import zio.actors.ActorSystemUtils
+import zio.actors.persistence.PersistenceId.PersistenceId
+import zio.actors.persistence.journal.Journal
+import zio.actors.persistence.jdbc.JDBCConfig.DbConfig
+import zio.blocking.Blocking
+import zio.interop.catz._
+
+private[actors] object JDBCJournal {
+
+  private lazy val runtime           = new DefaultRuntime {}
```

Comment:
Why do you need these?

Reply:
I think at the moment that this is required by the nature of ActorSystem.

We have such statements:

1. The journal is retrieved by each persistent actor during actor's instantiation.
2. As decided `ActorSystem` has no knowledge of concrete instances of `Stateful` (so whether actor is persistent or not).
3. There should be only one transactor for all journals that will use it

The result we need to have one transactor that ActorSystem has no knowledge of and that can be used by actors. 

To achieve it I decided to use `lazy val runtime.unsafeRun` so when actors arrive they already find single instance of transactor that can be shared. 

I thought about it a bit and I couldn't find another solution without involving `ActorSystem`

What do you think about it?